src/test/java/uk/gov/pay/adminusers/utils/email/EmailValidatorIsPublicSectorEmailTest.java

```diff
@@ -83,6 +83,7 @@ public EmailValidatorIsPublicSectorEmailTest(String email, boolean testResult) {
                 {"valid@slc.co.uk", true},
                 {"valid@socialworkengland.org.uk", true},
                 {"valid@ucds.email", true},
+                {"valid@wmca.org.uk", true},
```

Comment:
I think you are missing `{"valid@subdomain.wmca.org.uk", true}` test?

Reply:
i'm not sure but i can add it anyway.

Reply:
I am not sure about org.apache.commons.validator.routines.EmailValidator#EMAIL_REGEX as well, but we have subdomain tests just in case